shell/platform/darwin/ios/framework/Source/FlutterPlatformViews_Internal.mm

```diff
@@ -70,6 +76,77 @@ - (BOOL)pointInside:(CGPoint)point withEvent:(UIEvent*)event {
   return NO;
 }
 
+// Creates and initializes a UIVisualEffectView with a UIBlurEffect. Extracts and returns its
+// gaussianFilter. Logs errors and returns if Apple's API has changed and the filter can't be
+// extracted.
+- (NSObject*)extractGaussianFilter {
+  UIVisualEffectView* visualEffectView = [[UIVisualEffectView alloc]
+      initWithEffect:[UIBlurEffect effectWithStyle:UIBlurEffectStyleLight]];
+
+  NSObject* gaussianFilter =
+      [[[visualEffectView.subviews firstObject].layer.filters firstObject] retain];
```

Comment:
Does it make sense to traverse the entire layer tree rooted at `visualEffectView.layer` to find the first occurance of a gaussian blur filter so it's a little more robust?

Reply:
Currently there are only two filters in the layer tree so the traversal would be fast. My thinking was that if any part of the API changes, we should check that the filter still works, but it is possible for the gaussianBlur filter to keep its implementation even if the filters implementation changes. I think it makes sense.  

If the gaussian blur filter is in a different index,